MastodonSDK/Sources/MastodonUI/Scene/ComposeContent/Poll/PollOptionRow.swift

```diff
@@ -38,12 +43,67 @@ public struct PollOptionRow: View {
                     viewModel.shouldBecomeFirstResponder = false
                     viewModel.textField?.becomeFirstResponder()
                 }
+
+                if #available(iOS 16.0, *) {
+                    field.accessibilityActions {
+                        if let moveUp {
+                            Button(L10n.Scene.Compose.Poll.moveUp, action: moveUp)
+                        }
+                        if let moveDown {
+                            Button(L10n.Scene.Compose.Poll.moveDown, action: moveDown)
+                        }
+                        if let removeOption {
+                            Button(L10n.Scene.Compose.Poll.removeOption, action: removeOption)
+                        }
+                    }
+                } else {
+                    // beautiful!
```

Comment:
@j-f1 I allowed myself to refactor this, I hope that's ok for you, it's not meant offensive I just also didn't want to consume your time too much as you're already doing a fabulous job.

Reply:
That would be great! I couldn’t think of a better way to do this, would love to see what you came up with